spec/Body/StaticLights.vspec

```diff
@@ -0,0 +1,12 @@
+# (C) 2022 BMW Group - All rights reserved.
+# AUTHOR: Gerrit Grotewold BMW Group;
```

Comment:
We have previously not included "Author" in VSS files. I do not really have a strong opinion but I think a short discussion could be useful. Do we want "author" to be included, if so shall it be optional, recommended or mandatory going forward. If we want to use "author" I assume it will be handled like copyright, over time additional names will be added based on additions to the file.

Reply:
Meeting notes: remove copyright